src/services/staticPropsFetcher.ts

```diff
@@ -0,0 +1,15 @@
+import { dehydrate, QueryClient } from 'react-query';
+
+export const staticPropsFetcher = async (
+  key: string | (string | number)[],
+  fetchFunc: () => void,
+) => {
+  const queryClient = new QueryClient();
+  await queryClient.prefetchQuery(key, fetchFunc);
+
+  return {
+    props: {
+      dehydratedState: dehydrate(queryClient),
```

Comment:
Dit zou denk ik prima `state` kunnen heten. Het is niet heel belangrijk om expliciet aan te geven dat de state is ge-dehydrate (scheelt ook typen haha)